app/views/browse/route.html

```diff
@@ -56,15 +56,15 @@
                   <dt>Hostname<span ng-if="route.status.ingress.length > 1">s</span>:</dt>
                   <dd>
                     <span ng-if="!route.status.ingress">
-                      {{route | routeLabel}}
+                      {{route | routeLabelWithNoPath}}
```

Comment:
Assuming you change routeLabel to take in omitPath then this would just become `route | routeLabel : true`